### DISCUSSION TITLE
More organised documentation

### DISCUSSION THREAD

**Discussion by debugger24** [actions/actions-runner-controller#1722] [Polls]
For our docs currently, our README has become quite large and will grow even more in future. Should we create something more organized and interactive like [jupyterbooks](https://jupyterbook.org/en/stable/intro.html) (or something similar, feel free to add some alternative suggestions)? We can host it on GH Pages.

**Comment by toast-gear** [comment]
We have plans to move the docs over to MkDocs hosted in the existing gh-pages site, this looks similar? I don't have especially strong views on MkDocs or not, it's just one of the static site generators that I've used and was alright.

**Reply by debugger24** [reply]
Yes, it's similar to jupyterbooks. It uses sphinx behind the scene. I used both jupyterbooks and sphinx for documentation in past projects, really liked it.

Don't have a strong opinion on one over the other.
